protocols/identify/src/behaviour.rs

```diff
@@ -118,7 +120,7 @@ pub struct Config {
     /// e.g. `ipfs/1.0.0` or `polkadot/1.0.0`.
     protocol_version: String,
     /// The public key of the local node. To report on the wire.
```

Comment:
Should we update the comment to reflect on `CryptoKey`?

Reply:
Yes the documentation hasn't been updated, will do.

Reply:
Also the name `CryptoKey` is a weird name but I don't have a better idea, any thoughts?